MobProgrammingFacilitorsGuide_French.md

```diff
@@ -1,214 +1,248 @@
-# 1st Time Mob Programming Facilitor's Guide
-By [@LlewellynFalco](https://twitter.com/llewellynfalco)
+# Guide de facilitation pour une première session de Mob Programming
+Par [@LlewellynFalco](https://twitter.com/llewellynfalco)
 
+Ce document est un petit guide pour vous accompagner à la facilitation de votre première session de Mob Programming avec votre équipe.
+Faciliter un Mob est une compétence à part entière, et ne nécessite pas d'expertise technique préalable.
 
-This is a short guide to facilitating your 1st Mob Programming session with your team. Facilitating a mob is a separate skill, and does not require any prior technical expertise. 
+## Choisir le problème
+L'objectif au début, devrait être de réaliser quelque chose de simple et de s'amuser en apprenant le Mob Programming. Ainsi, vous devriez commencer par un problème simple. 
 
-## Picking the problem  
-The goal in the beginning should be to do something simple 
-and have fun learning to mob. As such, you should start with an easy problem.   
- 
-How do you know if the problem is easy enough?  
-Ask your developers if they believe they can solve this problem alone in under an hour. 
+Comment savoir si un problème est suffisamment simple ?
+Demandez à vos développeurs s'ils pensent résoudre le problème, seuls, en moins d'une heure.
 
-If you are new to mobbing, it's best to start with a simple exercise, not a work task.
-It is ok to take a simple task from your workload, just start very small. No, smaller.
+Si vous êtes nouveau dans la pratique du mob programming, il est plus approprié de commencer par un exercice, et non par une tâche de production.
+Néanmoins, c'est possible de prendre une tâche simple de votre encourt, commencez petit. Encore plus petit. 
 
-If you want a simple exercise, [The FizzBuzz kata](http://codingdojo.org/kata/FizzBuzz/) is a rather good starting point.
+Si vous cherchez un exercice simple, le [Kata The FizzBuzz](http://codingdojo.org/kata/FizzBuzz/) reste un bon point de départ. 
```

Comment:
ôter le "The"
si tu veux coller à la structure originale, mettre "le" dans les crochets (mais j'aime autant ce que tu as fait)

"constitue" plutôt que "reste"

aucune règle de typographie ne t'oblige à mettre un majuscule à "kata" ;)